Stripe/STPPaymentCardTextField.m

```diff
@@ -1106,7 +1114,12 @@ must be visible so they can be tapped over to (although
 - (STPFormTextField *)buildTextField {
     STPFormTextField *textField = [[STPFormTextField alloc] initWithFrame:CGRectZero];
     textField.backgroundColor = [UIColor clearColor];
-    textField.keyboardType = UIKeyboardTypePhonePad;
+    // setCountryCode: updates the postalCodeField keyboardType, this is safe
+    if (@available(iOS 10, *)) {
+        textField.keyboardType = UIKeyboardTypeASCIICapableNumberPad;
+    } else {
+        textField.keyboardType = UIKeyboardTypeNumberPad;
```

Comment:
should this be `PhonePad`?

Reply:
Yes, it should be, thanks for catching that!